### PR TITLE
Fix audio for video recordings for chrome

### PR DESCRIPTION
**Pull request by misi**
* Use the default working and config dir ~/.config/pulse
* Add global pulse client.conf the cookie

### CODE REVIEW

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: static/chrome/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,8 @@ COPY entrypoint.sh /
 RUN \
     chmod +x /usr/bin/chromedriver && \
     chown -R selenium:nogroup /etc/opt/chrome/policies/managed/
+
+RUN echo "cookie-file = ~/.config/pulse/cookie" >> /etc/pulse/client.conf
```
[the remark this answers is not in the document]
Comment (misi) [reply static/chrome/Dockerfile]: Fixed in bd76c3c

File: static/firefox/selenoid/Dockerfile [hunk 2]
```diff
@@ -8,5 +8,7 @@ COPY entrypoint.sh /
 
 USER selenium
 
+RUN echo "cookie-file = ~/.config/pulse/cookie" >> /etc/pulse/client.conf
```
Comment (vania-pooh) [inline static/firefox/selenoid/Dockerfile:11]: Here  you are creating config file as user `selenium`, whereas for chrome, opera and yandex - under `root`. Shouldn't this behavior be the same?

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: static/firefox/selenoid/Dockerfile (hunk 2 above)
Comment (misi) [reply static/firefox/selenoid/Dockerfile:9]: Yes sorry, my bad.  Great catch! Fixed it in 4349064. :pray:

**Review by vania-pooh — APPROVED** [review]

**Comment by vania-pooh** [comment]
@misi squashed and merged. thank you for contribution!

**Pull request merged**